doc/api/documentation.md

```diff
@@ -42,7 +42,7 @@ The stability indices are as follows:
 
 > Stability: 3 - Legacy. The feature is no longer recommended for use. While it
 > likely will not be removed, and is still covered by semantic-versioning
-> guarantees, use of the feature should be avoided.
+> guarantees, use of the feature should be avoided if possible.
```

Comment:
I'd prefer to keep the stronger wording. In my experience people will ignore stronger warnings when it's not possible, but will pick up something that's got a softer warning effectively because of logic along the lines of"how bad could it be".

Reply:
We could even strengthen it further by making it active voice. (Suggestion to follow....)

Reply:
I too would prefer the stronger wording if we were okay with emitting a runtime warning (this is not the same as removing the API), which [people seem to be -1 about](https://github.com/nodejs/node/issues/42232#issuecomment-1059987413).